guide/multiline.md

```diff
@@ -0,0 +1,92 @@
+# Multiline Text
+
+Text can span multiple lines as long as it is indented by at least one space.
+Line breaks inside of multiline text are preserved in the final text value.
+
+```
+single = Text can be written in a single line.
+
+multi = Text can also span multiple lines
+    as long as each new line is indented
+    by at least one space.
+
+block =
+    Sometimes it's more readable to format
+    multiline text as a "block", which means
+    starting it on a new line. All lines must
+    be indented by at least one space.
+```
+
+In almost all cases, patterns start at the first non-blank character and
+end at the last non-blank character. In other words, the leading and
+trailing blanks are ignored. There's one exception to this rule, due to
+another rule; see the `multiline2` message below.
+
+```
+leading-spaces =     This message's value starts with the word "This".
+leading-lines =
+
+
+    This message's value starts with the word "This".
+    The blank lines under the identifier are ignored.
+```
+
+In multiline patterns, the maximum indent common to all indented lines is
+ignored when the text value is spread across multiple indented lines.
+
+```
+multiline1 =
+    This message has four spaces of indent
+        on the second line of its value.
+```
+
+You can visualize this behavior in the following manner:
+
+```
+# █ denotes the indent common to all lines (removed from the value).
+# · denotes the indent preserved in the final value.
+multiline1 =
+████This message has four spaces of indent
+████····on the second line of its value.
+```
+
+This behavior also applies when the first line of a text block is indented
+relative to the following lines. This is the only case where a sequence of
+leading blank characters might be preserved as part of the text value, even
+if they're technically in the leading position.
+
+```
+multiline2 =
+        This message has four spaces of indent
```

Comment:
Is it worth to use the same visual clues used in multiline1 above? I find them very helpful.